lib/internal/Magento/Framework/Config/Dom.php

```diff
@@ -274,7 +274,8 @@ public static function validateDomDocument(
         $schema = self::$urnResolver->getRealPath($schema);
         libxml_use_internal_errors(true);
         try {
-            if (file_exists($schema)) {
+            // 4096 is the maximum length allowed by file_exists for the filename
+            if (strlen($schema) < 4096 && file_exists($schema)) {
```

Comment:
I mean it's already removed and will be published with next publication. Merging this PR will just cause extra conflict.